Syntax error in SQL statement in Business Intelligence service during start

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2021-12-19 14:01:29.030 ERROR 47302 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.springframework.boot.autoconfigure.web.servlet.WebMvcAutoConfiguration$EnableWebMvcConfiguration': Unsatisfied dependency expressed through method 'setConfigurers' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'openEntityManagerInViewInterceptorConfigurer' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/JpaBaseConfiguration$JpaWebConfiguration.class]: Unsatisfied dependency expressed through method 'openEntityManagerInViewInterceptorConfigurer' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'openEntityManagerInViewInterceptor' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/JpaBaseConfiguration$JpaWebConfiguration.class]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [com/abixen/platform/service/businessintelligence/infrastructure/configuration/BusinessIntelligenceServiceLiquibaseConfiguration.class]: Invocation of init method failed; nested exception is liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set liquibase/changelog/multi-visualisation/data-source-value-date-changelog.xml::data-value-date-v1::MariuszKumor:
     Reason: liquibase.exception.DatabaseException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202] [Failed SQL: (42001) CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL)]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.resolveMethodArguments(AutowiredAnnotationBeanPostProcessor.java:767) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.inject(AutowiredAnnotationBeanPostProcessor.java:719) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:119) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:399) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1431) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:410) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1352) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1195) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:213) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.context.event.AbstractApplicationEventMulticaster.retrieveApplicationListeners(AbstractApplicationEventMulticaster.java:264) ~[spring-context-5.3.13.jar:5.3.13]
	at org.springframework.context.event.AbstractApplicationEventMulticaster.getApplicationListeners(AbstractApplicationEventMulticaster.java:221) ~[spring-context-5.3.13.jar:5.3.13]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:138) ~[spring-context-5.3.13.jar:5.3.13]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:131) ~[spring-context-5.3.13.jar:5.3.13]
	at org.springframework.context.support.AbstractApplicationContext.registerListeners(AbstractApplicationContext.java:881) ~[spring-context-5.3.13.jar:5.3.13]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:580) ~[spring-context-5.3.13.jar:5.3.13]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:145) ~[spring-boot-2.6.1.jar:2.6.1]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:730) ~[spring-boot-2.6.1.jar:2.6.1]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:412) ~[spring-boot-2.6.1.jar:2.6.1]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:302) ~[spring-boot-2.6.1.jar:2.6.1]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1301) ~[spring-boot-2.6.1.jar:2.6.1]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1290) ~[spring-boot-2.6.1.jar:2.6.1]
	at com.abixen.platform.service.businessintelligence.BusinessIntelligenceServiceApplication.main(BusinessIntelligenceServiceApplication.java:38) ~[classes/:na]
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'openEntityManagerInViewInterceptorConfigurer' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/JpaBaseConfiguration$JpaWebConfiguration.class]: Unsatisfied dependency expressed through method 'openEntityManagerInViewInterceptorConfigurer' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'openEntityManagerInViewInterceptor' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/JpaBaseConfiguration$JpaWebConfiguration.class]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [com/abixen/platform/service/businessintelligence/infrastructure/configuration/BusinessIntelligenceServiceLiquibaseConfiguration.class]: Invocation of init method failed; nested exception is liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set liquibase/changelog/multi-visualisation/data-source-value-date-changelog.xml::data-value-date-v1::MariuszKumor:
     Reason: liquibase.exception.DatabaseException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202] [Failed SQL: (42001) CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL)]
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:800) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:541) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1352) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1195) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.addCandidateEntry(DefaultListableBeanFactory.java:1598) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1562) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveMultipleBeans(DefaultListableBeanFactory.java:1451) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1338) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1300) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.resolveMethodArguments(AutowiredAnnotationBeanPostProcessor.java:759) ~[spring-beans-5.3.13.jar:5.3.13]
	... 32 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'openEntityManagerInViewInterceptor' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/JpaBaseConfiguration$JpaWebConfiguration.class]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [com/abixen/platform/service/businessintelligence/infrastructure/configuration/BusinessIntelligenceServiceLiquibaseConfiguration.class]: Invocation of init method failed; nested exception is liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set liquibase/changelog/multi-visualisation/data-source-value-date-changelog.xml::data-value-date-v1::MariuszKumor:
     Reason: liquibase.exception.DatabaseException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202] [Failed SQL: (42001) CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL)]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:628) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1380) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1300) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:887) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791) ~[spring-beans-5.3.13.jar:5.3.13]
	... 48 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [com/abixen/platform/service/businessintelligence/infrastructure/configuration/BusinessIntelligenceServiceLiquibaseConfiguration.class]: Invocation of init method failed; nested exception is liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set liquibase/changelog/multi-visualisation/data-source-value-date-changelog.xml::data-value-date-v1::MariuszKumor:
     Reason: liquibase.exception.DatabaseException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202] [Failed SQL: (42001) CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL)]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1804) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:233) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveNamedBean(DefaultListableBeanFactory.java:1273) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveNamedBean(DefaultListableBeanFactory.java:1234) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveBean(DefaultListableBeanFactory.java:494) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:349) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBean(DefaultListableBeanFactory.java:342) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.orm.jpa.EntityManagerFactoryUtils.findEntityManagerFactory(EntityManagerFactoryUtils.java:121) ~[spring-orm-5.3.13.jar:5.3.13]
	at org.springframework.orm.jpa.EntityManagerFactoryAccessor.setBeanFactory(EntityManagerFactoryAccessor.java:155) ~[spring-orm-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeAwareMethods(AbstractAutowireCapableBeanFactory.java:1826) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1791) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.13.jar:5.3.13]
	... 58 common frames omitted
Caused by: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set liquibase/changelog/multi-visualisation/data-source-value-date-changelog.xml::data-value-date-v1::MariuszKumor:
     Reason: liquibase.exception.DatabaseException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202] [Failed SQL: (42001) CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL)]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:124) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:265) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.lambda$child$0(Scope.java:177) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:186) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:176) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:155) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:240) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:264) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.lambda$child$0(Scope.java:177) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:186) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:176) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:155) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Liquibase.runInScope(Liquibase.java:2404) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Liquibase.update(Liquibase.java:211) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Liquibase.update(Liquibase.java:197) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:314) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:269) ~[liquibase-core-4.6.2.jar:na]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863) ~[spring-beans-5.3.13.jar:5.3.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800) ~[spring-beans-5.3.13.jar:5.3.13]
	... 76 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set liquibase/changelog/multi-visualisation/data-source-value-date-changelog.xml::data-value-date-v1::MariuszKumor:
     Reason: liquibase.exception.DatabaseException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202] [Failed SQL: (42001) CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL)]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:695) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:49) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:111) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.lambda$child$0(Scope.java:177) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:186) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:176) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:155) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:110) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.lambda$child$0(Scope.java:177) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:186) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:176) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:155) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:240) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:94) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.lambda$child$0(Scope.java:177) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:186) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:176) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:155) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:240) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.Scope.child(Scope.java:244) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:66) ~[liquibase-core-4.6.2.jar:na]
	... 94 common frames omitted
Caused by: liquibase.exception.DatabaseException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202] [Failed SQL: (42001) CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL)]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:397) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:83) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:151) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1279) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1261) ~[liquibase-core-4.6.2.jar:na]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:660) ~[liquibase-core-4.6.2.jar:na]
	... 114 common frames omitted
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PUBLIC.DATA_VALUE_DATE (VALUE[*] TIMESTAMP, ID BIGINT NOT NULL)"; expected "identifier"; SQL statement:
CREATE TABLE PUBLIC.data_value_date (value TIMESTAMP, id BIGINT NOT NULL) [42001-202]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.message.DbException.getSyntaxError(DbException.java:261) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.readIdentifier(Parser.java:6080) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:10393) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.parseCreateTable(Parser.java:10336) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.parseCreate(Parser.java:7803) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.parsePrepared(Parser.java:1140) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.parse(Parser.java:1068) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.parse(Parser.java:1037) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.command.Parser.prepareCommand(Parser.java:965) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:614) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:552) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237) ~[h2-2.0.202.jar:2.0.202]
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223) ~[h2-2.0.202.jar:2.0.202]
	at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:94) ~[HikariCP-4.0.3.jar:na]
	at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java) ~[HikariCP-4.0.3.jar:na]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:393) ~[liquibase-core-4.6.2.jar:na]
	... 119 common frames omitted

--- a/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueDate.java
+++ b/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueDate.java
@@ -30,7 +30,7 @@ public final class DataValueDate extends DataValue<Date> {
 
     private static final long serialVersionUID = -655187403228089633L;
 
-    @Column(name = "value", nullable = true)
+    @Column(name = "value_", nullable = true)
     private Date value;
 
     private DataValueDate() {

--- a/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueDouble.java
+++ b/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueDouble.java
@@ -30,7 +30,7 @@ public final class DataValueDouble extends DataValue<Double> {
 
     private static final long serialVersionUID = -1434303122608329824L;
 
-    @Column(name = "value", nullable = true)
+    @Column(name = "value_", nullable = true)
     private Double value;
 
     private DataValueDouble() {

--- a/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueInteger.java
+++ b/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueInteger.java
@@ -30,7 +30,7 @@ public final class DataValueInteger extends DataValue<Integer> {
 
     private static final long serialVersionUID = -4059158260855268805L;
 
-    @Column(name = "value", nullable = true)
+    @Column(name = "value_", nullable = true)
     private Integer value;
 
     private DataValueInteger() {

--- a/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueString.java
+++ b/abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/data/DataValueString.java
@@ -33,7 +33,7 @@ public final class DataValueString extends DataValue<String> {
     private static final long serialVersionUID = 4578637099421599970L;
 
     @Length(min = 0, max = DATASOURCE_VALUE_STRING_MAX_LENGTH)
-    @Column(name = "value", length = DATASOURCE_VALUE_STRING_MAX_LENGTH)
+    @Column(name = "value_", length = DATASOURCE_VALUE_STRING_MAX_LENGTH)
     private String value;
 
     private DataValueString() {

--- a/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-date-changelog.xml
+++ b/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-date-changelog.xml
@@ -8,7 +8,7 @@
 
     <changeSet author="MariuszKumor" id="data-value-date-v1">
         <createTable tableName="data_value_date">
-            <column name="value" type="timestamp"/>
+            <column name="value_" type="timestamp"/>
             <column name="id" type="bigint">
                 <constraints nullable="false"/>
             </column>

--- a/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-double-changelog.xml
+++ b/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-double-changelog.xml
@@ -8,7 +8,7 @@
 
     <changeSet author="MariuszKumor" id="data-value-double-v1">
         <createTable tableName="data_value_double">
-            <column name="value" type="double"/>
+            <column name="value_" type="double"/>
             <column name="id" type="bigint">
                 <constraints nullable="false"/>
             </column>

--- a/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-integer-changelog.xml
+++ b/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-integer-changelog.xml
@@ -8,7 +8,7 @@
 
     <changeSet author="MariuszKumor" id="data-value-integer-v1">
         <createTable tableName="data_value_integer">
-            <column name="value" type="integer"/>
+            <column name="value_" type="integer"/>
             <column name="id" type="bigint">
                 <constraints nullable="false"/>
             </column>

--- a/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-string-changelog.xml
+++ b/abixen-platform-business-intelligence-service/src/main/resources/liquibase/changelog/multi-visualisation/data-value-string-changelog.xml
@@ -8,7 +8,7 @@
 
     <changeSet author="MariuszKumor" id="data-value-string-v1">
         <createTable tableName="data_value_string">
-            <column name="value" type="varchar(32)"/>
+            <column name="value_" type="varchar(32)"/>
             <column name="id" type="bigint">
                 <constraints nullable="false"/>
             </column>

